packages/react-avatar/src/components/Avatar/Avatar.types.ts

```diff
@@ -3,16 +3,21 @@ import { ComponentPropsCompat, ComponentStateCompat, ShorthandPropsCompat } from
 import { PresenceBadgeProps, PresenceBadgeStatus } from '@fluentui/react-badge';
 
 export interface AvatarProps extends ComponentPropsCompat, React.HTMLAttributes<HTMLElement> {
-  /** The Avatar's image. */
+  /**
+   * The Avatar's image.
+   */
   image?: ShorthandPropsCompat<React.ImgHTMLAttributes<HTMLImageElement>>;
 
   /**
-   * The label shown when there's no image or icon. Defaults to the initials derived from `name` using
-   * `getInitials`.
+   * The label shown when there's no image. Defaults to the initials derived from `name` using `getInitials`.
    */
   label?: ShorthandPropsCompat<React.HTMLAttributes<HTMLElement>>;
 
-  /** Icon displayed when there's no image. */
+  /**
+   * Icon to be displayed when the avatar doesn't have an image or name (or if getInitials returns an empty string).
+   *
+   * @defaultvalue `Person20Regular` (the default icon's size depends on the Avatar's size)
```

Comment:
`@defaultvalue` is invalid tag

```suggestion
   * @defaultValue `Person20Regular` (the default icon's size depends on the Avatar's size)
```

Reply:
It's written `defaultvalue` in all of react-avatar. I'll fix it up with #19259